docs/process/okrs.md

```diff
@@ -18,7 +18,7 @@ At Flexpa, OKRs are scored using [bankers rounding](https://wiki.c2.com/?Bankers
 * **0.4 to 0.6**: Yellow - Represents progress made but falling short of completion.
 * **0.7 to 1.0**: Green - Signifies successful delivery and achievement of the objective.
 
-When establishing OKRs, we adopt a scientific approach to avoid setting unattainable goals, where achieving a perfect score of 1.0 seems impossible or unrealistic. Each Objective and its corresponding Key Results are assigned a dedicated Flexpal as an Owner. The Owner is not solely responsible for the success of the Objective or Key Result. At Flexpa, we embrace a collaborative approach, and any Flexpal, regardless of their title or role, has the opportunity to be an Owner and contribute to the achievement of the OKRs.
+When establishing OKRs, we adopt a scientific approach to avoid setting unattainable goals, where achieving a perfect score of 1.0 seems impossible or unrealistic. Each Objective and its corresponding Key Results are assigned a dedicated Flexpal as an Owner. The Owner is not solely responsible for the success of the Objective or Key Result. At Flexpa, we embrace a collaborative approach, and any Flexpal, regardless of their title or role, has the opportunity to be an Owner and contribute to the achievement and planning of the OKRs. We maintain a document titled "Flexpa OKRs" that is always open to be viewed, commented on, and edited at any point in time and by any Flexpal. We encourage all Flexpals to think about OKRs throughout each quarter with a critical lens. We specifically encourage discussion about upcoming OKRs in the lead up to, and during, each quarterly retreat.
```

Comment:
In the flow of the document, it makes more sense to note that they are established at retreats (and link), prior to explaining detail about establishing them.

Further, unless the OS directly links to a document, it should describe the document instead of referring to it by title as a style guideline.

```suggestion
OKRs are established at [quarterly retreats](./retreats.md). In advance of the retreat, we prepare and share a draft. We specifically encourage discussion about upcoming OKRs in the lead up to, and during, each quarterly retreat.

When establishing OKRs, we adopt a scientific approach to avoid setting unattainable goals, where achieving a perfect score of 1.0 seems impossible or unrealistic. Each Objective and its corresponding Key Results are assigned a dedicated Flexpal as an Owner. The Owner is not solely responsible for the success of the Objective or Key Result. At Flexpa, we embrace a collaborative approach, and any Flexpal, regardless of their title or role, has the opportunity to be an Owner and contribute to the achievement and planning of the OKRs. 
```